packages/shared/src/lib/featureManagement.ts

```diff
@@ -130,6 +130,13 @@ export class Features {
     ['control', 'v1', 'v2', 'v3'],
   );
 
+  static readonly AuthVersion = new Features('auth_version', 'v1', [
```

Comment:
Check the way of implementation of `AdditionalInteractionButton` below introduced by Chris (in `featureValues.ts` you will have an enum `AuthVersions` that will be used here instead of strings)